markdown/2017-10-16.md

```diff
@@ -0,0 +1,180 @@
+acquired membership before 18.00 on the day of the event. Strictly 18+ only.
+
+---
+
+# Autumn Elections
+
+<center>
+<h3>**VOTING IS OPEN**. [Please vote](https://vote.union.ic.ac.uk/).</h3>
+</center>
+
+This is an opportunity to make your voice heard within the College and the
+Union.
+
+Every year students vote for at least one Academic Year Representative who
+will receive and collate feedback from their cohort and will pass this
+information to the Departmental Representatives or staff as appropriate.
+Academic Representatives have made long-lasting improvements in many areas
+such as the content of curricula, feedback quality and timeliness, exam
+timetables and formats, online learning resources, and the physical
+facilities available.
+ 
+This year the Union has also newly launched a Wellbeing Representation
+Network, which aims to tackle issues such as stress and pressure, mental
+health, the quality of personal tutoring, how students are supported to be
+resilient and independent, transitioning back into study after time off, and
+more. Wellbeing Representatives will work with College staff, Welfare
+Officers, and Academic Representatives.
+
+There are also candidates running to be a Union Council Ordinary Member, a
+position that involves proposing and agreeing Imperial College Union's
+policies on critical issues such as tuition fees, widening access, and the
+environment.
+
+You have until noon on Friday of this week to vote in the elections.
+
+Vote on the [Union voting page](https://vote.union.ic.ac.uk/), or read [more about the election](https://www.imperialcollegeunion.org/your-union/autumn-elections-2017).
+
+# CGCU Football League Team Entries Open!
+
+Hey Guys!
+
+Team entries are now open for the CGCU Football Championship!
+
+Get your friends together and represent your department for a chance to win
+the coveted CGCU Football Cup and be proclaimed CGCU Football Champions!
+Teams are 7-a-side and all equipment will be provided. Deadlines will be
+provided for when matches need to be played by, so your teams can organise
+your own matches for whenever suits you best!
+
+To enter: get a team of 7, name it and choose a captain. Submit these details
+and the captain's email to either myself or a member of your DepSoc
+committee.
+ 
+ENTRY IS FREE!!! My email for submissions is <asd15@ic.ac.uk>.
+ 
+DEADLINE FOR ENTRY: 1/11/17
+ 
+I hope to hear from as many of you as possible very soon - May the Best Team Win!
+
+Andrew Duckworth
+CGCU Sports Officer
+
+# ICSM Coding Looking for Volunteer Teaching Assistants
+
+ICSM Coding Society (brand new this year) aims to provide easy access to
+beginner-level coding tutorials for students with no previous coding
+experience. More info here:
+<https://www.imperialcollegeunion.org/activities/a-to-z/coding-medics> and
+<https://www.facebook.com/ICSMCoding/>
+
+We are running a series of Python coding workshops in November aimed at
```

Comment:
Can you quote all of this to make it clear that this wasn't written by docsoc

Reply:
I think I want to shorten it too, it's a bit too long. I'll make it sound like it's not coming from us. I don't think the current preprocessor and stylesheet supports blockquotes too well